### PR TITLE
Add a (mostly stubbed) blog entry for the "mapping consensus" article

### PR DESCRIPTION
**Pull request by dnadales**

### CODE REVIEW

**Comment by dnadales** [comment]
@abailly-iohk this is just a proposal, we can ditch the whole structure if needed. The idea is to get the discussion started and have a roadmap.

**Review by abailly-iohk — APPROVED** [review]
❤️

File: blog/mapping-consensus.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+## On the need for improving consensus documentation  
+
+<!-- Where were we and what were the problems -->
+
+Consensus was developed trying to meet strict deadlines: first IOG had to
+release the Byron rewrite as soon as possible, and later on, the release of
+Shelley became the next urgent thing. In spite of working under such high
+pressure, the consensus team did its best not to compromise on assurance,
+architecture, and documentation. A proof of the latter, is the [consensus
```
Comment (nfrisby) [inline blog/mapping-consensus.md:17]: > A proof of the latter, ...

Hmm. This sentence seems a bit inaccurate to me; Thomas and Edsko wrote that as they were leaving. The document didn't exist until after the Shelley hard fork happened. Thus this paragraph is currently not totally internally consistent.

**Comment by dnadales** [comment]
I'm closing this one since we are not going to move this forward in the foreseeable future.

**Pull request closed without merging**